src/models/poi/index.ts

```diff
@@ -29,3 +32,14 @@ export type Pois = Record<string, PoiData>;
 export type PoiOrNull = Poi | null;
 
 export default Poi;
+
+export interface PoiCategory {
+  type: string;
+  name: string;
+  descname: string;
```

Comment:
A name that fully express the meaning of the variable would be suggested.
No worry of the length of the variable. Also, please follow the lower camel case.

(correct me if I am wrong) If you are intended to use it as "descriptive name", I think `descriptiveName` would be a choice that you may want.

this suggestion also applies to `descname` in `PoiStatus`